metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -161,10 +184,13 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "KafkaSource":
 
     def get_workunits(self) -> Iterable[MetadataWorkUnit]:
         topics = self.consumer.list_topics().topics
-        for t in topics:
+
+        extra_topic_details = self.fetch_extra_topic_details(topics.keys())
```

Comment:
Surround the entire call with a try except

Reply:
maybe inside fetch_extra_topic_details

Reply:
yes